parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -475,7 +477,8 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	>;
 	type ControllerOriginConverter = xcm_config::XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
-	type PriceForSiblingDelivery = ();
+	type PriceForSiblingDelivery =
+		SiblingExponentialPrice<FeeAssetId, BaseDeliveryFee, TransactionByteFee, XcmpFeeFactor>;
```

Comment:
when https://github.com/paritytech/polkadot/pull/7005 and https://github.com/paritytech/polkadot/pull/7585,
I am goging to reuse here lots of stuff from `runtime/common` e.g. `ExponentialPrice / PriceForParachainDelivery ...`